src/actions/loadData.js

```diff
@@ -74,184 +75,118 @@ export const collectDatasetFetchUrls = (url) => {
     url = parts[0]; // eslint-disable-line no-param-reassign
     secondTreeUrl = parts[1];
   }
+  if (url.startsWith('/')) url = url.slice(1); // eslint-disable-line no-param-reassign
+  if (secondTreeUrl && secondTreeUrl.startsWith('/')) {
+    secondTreeUrl = secondTreeUrl.slice(1); // eslint-disable-line no-param-reassign
+  }
   return [url, secondTreeUrl];
 };
 
-/**
- * This is for processing a second tree using the deprecated
- * syntax of declaring second trees, e.g. `flu/seasonal/h3n2/ha:na/2y`
- * We are keeping this to allow backwards compatibility.
- *
- * given a url to fetch, check if a second tree is defined.
- * e.g. `ha:na`. If so, then we want to make two fetches,
- * one for `ha` and one for `na`.
- *
- * @returns {Array} [0] {string} url, modified as needed to represent main tree
- *                  [1] {string | undefined} secondTreeUrl
- *                  [2] {string | undefined} string of old syntax
- */
-const collectDatasetFetchUrlsDeprecatedSyntax = (url) => {
```

Comment:
Does the existing `collectDatasetFetchUrls` handle the deprecated syntax or does this mean we are dropping the ability to parse the deprecated syntax here?

Reply:
@jameshadfield it seems like in its current state this PR drops the ability to parse the deprecated second tree syntax, e.g. `/flu/seasonal/h3n2/ha:na/2y`, which works on the `master` branch. Is that intentional?